### ISSUE TITLE
Workshop should register in JEI as crafting table and as a furnace

### ISSUE THREAD

**Issue by Knito58** [TechReborn/EngineersWorkshopReborn#10]
###### **Forge Version & Engineer's Workshop Version [MAKE SURE YOU ARE RUNNING THE LATEST BUILD]**
> Forge 14.23.2.2624 EW 1.0.6.31

###### **Expected behavior**
> JEI shows plus sign and allows pushing of chosen recipe's items (from internal chest or players inv) into the crafting table

###### **Actual Behavior**
> Doh.

###### **Link to Crashlog or Forgelog** *[If Applicable]*
> Nothing happens - no crash.

###### **Steps to reproduce** *[If Applicable]*
> Use JEI and be disappointed.

Alot of other crafting tables are listed in JEI, why not EW? Maybe because the EW is not a crafting table, when no crafting table is in the upgrade slot? But this would not bother anybody as you always need a crafting table in the EW. Without JEI integration the vanilla crafting table is faster and the other advantages do not convince me to use it. I rather build a hoppered furnace aside my vanilla workbench and good it is. I really want something like EW but it does not offer speedy workflow yet.
Same as above is also true for the furnace part of EW.

I follow EW since 1.0.2.20 and thought this would heal over time. After 4 months of not using it I get over my natural laziness and tell you that: I like the idea, it is a great idea. But atm i do not like working with it. I do not want to throw with dirt - I really want to like it as the idea is great. But I'm so used to click the recipes in a crafting area with JEI that it feels as if it is broken if that does not work.

**Comment by modmuss50** [comment]
Yeah, thats a good idea. When I get some time I will look into it. It shouldn't be too hard to do.

**Comment by Knito58** [comment]
I'm very happy to read that and will stay tuned. :)
